packages/testing/src/api/index.ts

```diff
@@ -18,11 +18,14 @@ import { firebase } from '@firebase/app';
 import request from 'request-promise';
 import { FirebaseApp, FirebaseOptions } from '@firebase/app-types';
 import { base64 } from '@firebase/util';
+import { setLogLevel, LogLevel } from '@firebase/logger';
 
 /** The default url for the local database emulator. */
 const DBURL = 'http://localhost:9000';
 /** Passing this in tells the emulator to treat you as an admin. */
 const ADMIN_TOKEN = 'owner';
+/** Warnings aren't needed when product is in testing. */
```

Comment:
I'm curious what warnings exactly are you trying to suppress?  Normally I'd want a higher log level in testing, not lower...

Reply:
Ahhhhh...  Yep, I see what you mean there. :-)

Reply:
Thanks Ryan for the prompt response!